guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -39,21 +39,14 @@ ifdef::satellite[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos \
---enable={RepoRHEL9ServerSatelliteServerProjectVersion} \
---enable={RepoRHEL9ServerSatelliteMaintenanceProjectVersion} \
 --enable={RepoRHEL9BaseOS} \
 --enable={RepoRHEL9AppStream}
 ----
 endif::[]
 
 ifdef::katello[]
-. Install the `katello-repos-latest.rpm` package
+. Ensure that the {EL} 9 BaseOS and Appstream repositories are enabled.
```

Comment:
For upstream, this is now the same as EL 8 so at the very least we can drop them. I'd actually consider dropping the whole repository setup for upstream. Essentially you want a basic OS install which I'd feel confident in assuming from users.

Reply:
I'm fine with that.